canopy_main.ml

```diff
@@ -41,10 +41,10 @@ module Main  (C: CONSOLE) (S: STACKV4) (RES: Resolver_lwt.S) (CON: Conduit_mirag
     let module Store = Canopy_store.Store(C)(Context)(Inflator) in
     let open Canopy_utils in
     let cache = ref (KeyMap.empty) in
-    let open Canopy_config in
-    let config = config () in
+    Store.pull console >>= fun () ->
+    Store.fill_cache cache >>= fun _ ->
```

Comment:
I think it is a good idea to raise early with a missing uuid.  if we generate one on the fly, this will mess up caching of the feed.  rather include the generation in the `populate.sh` script (to be run for a fresh data repository)